acceptance/gossip_peerings_test.go

```diff
@@ -148,13 +160,21 @@ func TestGossipRestart(t *testing.T) {
 	checkRangeReplication(t, l, 10*time.Second)
 
 	log.Infof("stopping all nodes")
-	for _, node := range l.Nodes {
+	for i, node := range l.Nodes {
 		node.Stop(5)
+		expEvents = append(expEvents, []localcluster.Event{
+			{NodeIndex: i, Status: "kill"},
```

Comment:
that's awkward. It passes locally, so I can only assume that there's some difference in docker (i'm on the latest brew locally).